src/utils.js

```diff
@@ -0,0 +1,93 @@
+// user selects one of these options when first adding an item to a list
+export const radioButtonOptions = [
+  {
+    option: 'soon',
+    value: 7,
+    defaultChecked: true,
+  },
+  {
+    option: 'kind of soon',
+    value: 14,
+    defaultChecked: false,
+  },
+  {
+    option: 'not soon',
+    value: 30,
+    defaultChecked: false,
+  },
+];
+
+//used to normalize item names when added by a user
+export const removePunctuation = (string) => {
```

Comment:
looking at code, it seems this also removes spaces (split + join)? In that case, naming isn't as accurate.